test/resources/testproject/src/com/example/five.clj

```diff
@@ -58,6 +58,15 @@
     [:bar :foo]
     (count foo)))
 
+(when-let [{:keys [minor major]} *clojure-version*]
+  (and (>= major 1)
```

Comment:
this `and` is outside a condition

(if fixing this still doesn't work, then one would have to create a new `resources` dir just for this case. It would be added to :resource-paths only when using the :1.9 and :1.10 profiles)

Reply:
Thanks, still working on that first cup of coffee :)